paddle/phi/backends/onednn/onednn_reuse.h

```diff
@@ -1076,9 +1076,7 @@ class TransposeOneDNNHandler {
   std::shared_ptr<dnnl::memory> AcquireDstMemory(DenseTensor* output,
                                                  Place place) {
     auto dst_md = Axis2MemoryDesc(dims_, axis_);
-    output->Resize(make_ddim(dims_));
```

Comment:
I deleted Resize() and now it seems to work properly, @chenwhql @YuanRisheng please review

Reply:
done, please take a look at the Coverage CI @PaulinaGacek

Reply:
I have checked the Coverage CI in my machine and the code is exercised. I have marked it as successful.